styles/styles.css

```diff
@@ -346,6 +346,13 @@ main .section.highlight {
   background-color: var(--highlight-background-color);
 }
 
+/* FAQ page typography */
```

Comment:
This file is owned by the portal; which means it changes quite often and is immutable for customers. If I may suggest you to create a new stylesheet say `gmo-styles.css`. Then you can link the new stylesheet with `head.html` (which is mutable)